domains/auth/Auth.API/Integration.Tests/Extensions/NSubstituteExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Grpc.Core;
+using Grpc.Core.Testing;
+using NSubstitute;
+using NSubstitute.Core;
+
+namespace Integration.Tests.Extensions;
```

Comment:
Move this to:
<ProjectReference Include="..\..\Shared\Testing\Testing.csproj" />

Shared?

Reply:
I felt it was a bit much to make a new project just for this. But I see your point - want me to move it to shared project?

Reply:
It's been moved to Testing project